google-cloud-logging/src/main/java/com/google/cloud/logging/Severity.java

```diff
@@ -26,6 +26,9 @@
  */
 public enum Severity {
 
+  /** The None severity level. Should not be used with log entries. */
+  NONE(LogSeverity.UNRECOGNIZED),
+
```

Comment:
Unfortunately, the `Severity` constructor accepts only `LogSeverity` values... Not sure how else to solve rather than adding another constructor - let me add it then.

Reply:
I'm still a bit concerned about using UNRECOGNIZED for this, in case we need to include UNRECOGNIZED as a true option for this enum in the future. But assuming that UNRECOGNIZED exists only in the java protos and not in the proto definitions, it should be ok